src/_includes/layouts/pattern-library.hbs

```diff
@@ -4,43 +4,85 @@ styles:
 - /styles/eleventy-patterns/index
 scripts:
 - /scripts/eleventy-patterns/index
+- /scripts/eleventy-patterns/navigation
 ---
 <div class="EP_layout">
-  <aside class="EP_layout__sidebar">
-    <nav class="EP_nav">
+  <div class="EP_header-container">
+    <header class="EP_header">
       <h2>
-        <a href="/">
+        <a href="/" class="EP_nav__logo">
           Eleventy Patterns
         </a>
       </h2>
-
-      <div class="EP_nav__category">
-        <h3>
-          <a href="/prototypes">
+      <button class="EP_nav__menu-toggle EP_button" aria-expanded="false">Menu</button>
+    </header>
+    <nav class="EP_nav">
+      <ul class="EP_nav__menu">
+        <li class="EP_nav__category">
+          <a class="EP_nav__submenu-toggle" aria-expanded="false" role="button">
+              Prototypes
+            <svg xmlns="http://www.w3.org/2000/svg" width="20" height="10" viewBox="0 0 20 12.35" class="toggle-icon" aria-hidden="true" focusable="false">
```

Comment:
It seems odd that the width/height ratio on this SVG doesn't match the viewbox ratio.